app/assets/stylesheets/publisher.scss

```diff
@@ -29,19 +29,17 @@
     }
 
     .options_and_submit {
-      padding: 10px 0 ;
-      #publisher_service_icons {
-        .btn-link{ text-decoration: none; }
-        .btn-link.question_mark .entypo-cog {
+      padding: 10px 0;
+      #publisher-service-icons {
```

Comment:
Well rule about nesting here would mean I have to rewrite a big part of the SCSS :/